docker-compose.yaml

```diff
@@ -0,0 +1,24 @@
+version: "3.9"
+services:
+  graph_db:
+    container_name: ot_node_graph_db
+    image: khaller/graphdb-free:latest
+    ports:
+      - "7200:7200"
+    network_mode: host
+
+
+  ot_node_v6:
+    container_name: ot_node_v6
+    build: .
+    ports:
+      - "3306:3306"
+      - "8900:8900"
+      - "9000:9000"
+    command: >
+       bash -c "
+        service mysql restart && 
+        supervisord -c /ot-node/current/supervisord.conf                
+        "
+    network_mode: host
```

Comment:
Someone from community already tried out the docker image, and here is the comment he left "network_mode: Host and exposing Port will create an error using newer docker-compose versions", so please check this

Reply:
will only use network_mode then

Reply:
Also, let's use the latest docker compose version. Do we have any blockers there?

Reply:
That's the latest have you checked.

Reply:
Yup, hey that comment came from me.
Basically have been playing around docker and docker compose myself. And it seems using the latest docker_compose version **network_mode : host** automatically uses the port, which is specified and exposed to your localhost. Which is why you will have to specify either **- ports : .... or network_mode: host, but not both**

Seems like the latest version also deprecates/ignores depends_on, which I tried to circumvent the Node running into connection issues with the  not yet booted graph db.

Reply:
@UniMa007 Thanks for the info, we will proceed with network_mode:host only.

Reply:
Hey @UniMa007 thank you for the contribution!